src/http/api/resources/ping.js

```diff
@@ -23,9 +24,19 @@ exports.get = {
   handler: (request, reply) => {
```

Comment:
@alanshaw @hugomrdias @achingbrain et al, can I get your eyes to go through this handler. This stream tagliatelle is a nightmare came true.

Reply:
Stream tagliatelle - I’m going to use that one